app/src/main/java/com/example/mykotlincalculator21/mvp/model/CalculatorModel.kt

```diff
@@ -1,16 +1,59 @@
 package com.example.mykotlincalculator21.mvp.model
 
 import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
-import com.example.mykotlincalculator21.utils.EMPTY_STRING
+import com.example.mykotlincalculator21.utils.*
```

Comment:
try dont to use wild imports

Reply:
you can configure AS in File -> Settings
![image](https://user-images.githubusercontent.com/47530344/105363107-72aef980-5bda-11eb-9d7b-443e25fc9b66.png)

Reply:
Thx for the tip